src/redux/Actions/PortfolioActions.js

```diff
@@ -87,3 +85,22 @@ export const updatePortfolio = (portfolioData) => ({
     }
   }
 });
+
+export const removePortfolio = (portfolio) => ({
+  type: ActionTypes.REMOVE_PORTFOLIO,
+  payload: PortfolioHelper.removePortfolio(portfolio),
+  meta: {
+    notifications: {
+      fulfilled: {
+        variant: 'success',
+        title: 'Success removing portfolio',
+        description: 'The portfolio was removed successfully.'
+      },
+      rejected: {
```

Comment:
I think it might be better to let the notifications middleware to handle promise middleware errors. If we use this, we will hide the actual reason why the request failed from the user. The default error notification will show the error message from API.

Reply:
done